### ISSUE TITLE
Telcontar font: missing numerals

### ISSUE THREAD

**Issue by phoe** [arnog/tecendil-js#31]
![Zrzut ekranu z 2022-01-22 13-28-58](https://user-images.githubusercontent.com/15045546/150638555-496e4a8a-6566-4d31-9586-6f50d38c6426.png)

Firefox 95 on Linux. It seems that the numbers are written in a fallback font rather than Telcontar itself. I can also observe this in the Handbook.

**Comment by j-teuber** [comment]
You are right, [Telcontar doesn't have those numerals](http://freetengwar.sourceforge.net/html-files/tengtelc-discussion-008.pdf), John Winge simply didn't design those. There are other fonts like e. g. Annatar with numerals that would fit Telcontar better, but this is not actually a CSS fallback but the actual glyphs in Telcontar that Winge copy-pasted from FreeMono Tengwar, so to change it one would need to remove them from Telcontar or implement a possibility to switch the font mid-text (which is both possible, but I'm not sure if Arno is willing to do that at the moment).

**Comment by arnog** [comment]
It's possible to implement a fallback for some Unicode range in CSS, and I tried it at one point, but unfortunately it introduced other problems (which I don't recall at the moment...). If someone is handy with a Font editor program and were to submit a version of Telcontar with some better numerals (or could somehow convinced Johan (I've tried)), I'd be happy to update the font.
